Code/Engine/Foundation/Types/Implementation/Variant_inl.h

```diff
@@ -6,166 +6,199 @@ EZ_ALWAYS_INLINE ezVariant::ezVariant()
 }
 
 EZ_ALWAYS_INLINE ezVariant::ezVariant(const ezVariant& other)
+  : ezVariant()
```

Comment:
I don't understand this change. Seems like a complete waste to first default init all values and then overwrite them again with the proper values.

Reply:
Reverting this for now, as it has no benefit other than making some static analyzer happy. Need to follow up on why the analyzer complained about it.